### PR TITLE
Task/name mapping logs

### PR DESCRIPTION
**Pull request by AlvaroVega**
issue https://github.com/telefonicaid/fiware-cygnus/issues/1937

### CODE REVIEW

**Review by fgalan — APPROVED** [review]
LGTM

File: cygnus-ngsi/src/main/java/com/telefonica/iot/cygnus/interceptors/NGSINameMappingsInterceptor.java [hunk 1]
```diff
@@ -507,6 +509,7 @@ protected void loadNameMappings(String jsonStr) {
             LOGGER.debug("[nmi] newCA: " + newCA.toString());
         } // for
         LOGGER.debug("[nmi] newCE: " + newCE.toString());
+        LOGGER.info("[nmi] Entity " + originalCE.toString() + " mapped to: " + newCE.toString() + " by matched with " + entityMapping);
```
[the remark this answers is not in the document]
Comment (AlvaroVega) [reply cygnus-ngsi/src/main/java/com/telefonica/iot/cygnus/interceptors/NGSINameMappingsInterceptor.java]: fixed in https://github.com/telefonicaid/fiware-cygnus/pull/1961/commits/84bcb72329fc00c7d9d26e58c95899d3b8bdadaf

File: cygnus-ngsi/src/main/java/com/telefonica/iot/cygnus/interceptors/NGSINameMappingsInterceptor.java [hunk 2]
```diff
@@ -440,6 +441,7 @@ protected void loadNameMappings(String jsonStr) {
 
         if (entityMapping == null) {
             LOGGER.debug("[nmi] Entity not found: " + originalEntityId + ", " + originalEntityType);
+            LOGGER.info("[nmi] Entity " + originalCE.toString() + " no matched");
```
[the remark this answers is not in the document]
Comment (AlvaroVega) [reply cygnus-ngsi/src/main/java/com/telefonica/iot/cygnus/interceptors/NGSINameMappingsInterceptor.java:443]: fixed in https://github.com/telefonicaid/fiware-cygnus/pull/1961/commits/84bcb72329fc00c7d9d26e58c95899d3b8bdadaf

**Review by IvanHdzC — APPROVED** [review]
LGTM

**Pull request merged**